### ISSUE TITLE
[Textual] -  Policy

### ISSUE THREAD

**Issue by conconnl** [joomla-extensions/pkg_languagepack#22]
On the current landing page: https://community.joomla.org/translations.html
We have a link to:

* https://community.joomla.org/translations/joomla-project-translation-and-localization-policy.html
This page holds legal type of information as well duplication of existing policies maintained and updated on official pages like the TM website. At the same time it holds FAQs, definitions and other things.
We need to define how and where to put it on the downloads site and at the same time remove data listed on the right places (deduplication) update it accordingly to the current situation.

**Comment by conconnl** [comment]
@infograf768 what are your thoughts one this?
Where on the new location are we going to put the information which shouldn't be on the other official locations.

**Comment by infograf768** [comment]
This is what we have now
<img width="1304" alt="Screen Shot 2020-12-14 at 10 37 39" src="https://user-images.githubusercontent.com/869724/102065278-a38c5980-3df8-11eb-98e2-c7057651dbab.png">

Remark:
Reduce descriptions for 1.0, 1.5, 2.5 to just mark these as archives.

Concerning the Translation Policy, It could be a link at the bottom  of that language packs page as done on community now but it looks a bit weird to have this on the download site.

The duplication of information was done on purpose at the time as we wanted a full agreement of this policy by TTs coordinators without them picking stuff all over the place. 
We have maintained for each J version a spreadsheet with the TTs formal agreement since its implementation. Mig should have it and it has to be done again for J4.

For your info, I was unofficially part of the TM team at the time, in charge of making at least sure the sites concerned obeyed to the J rules, disclaimer and all.

So, basically, you folks decide...

**Comment by conconnl** [comment]
I have had a discussion with multiple people.
The conclusion from all of them is that the page should be in the Docs site, where the page of the team is located as well.
Because it has nothing to do with the downloads itself, but with the work, support and rules related to the team and translation process.

Off course a link on the download landing page to the team / information, to get translators to the right information.

**Comment by conconnl** [comment]
The docs article has been prepared with the current information and some tweaks.
I will see if I can get the translation team to make some updates on the document
https://docs.joomla.org/Translation_of_Language_Packs

**Comment by infograf768** [comment]
> I will see if I can get the translation team to make some updates on the document

Rather a work for the TM team, is'nt it?

**Comment by infograf768** [comment]
Also, beware:
If the Policy is in docs, it SHOULD NOT BE EDITABLE by anybody!!!! 
It is an official policy.

**Comment by conconnl** [comment]
> > I will see if I can get the translation team to make some updates on the document
> 
> Rather a work for the TM team, is'nt it?

Yes and No, it seems to hold multiple information.
As it's a volunteer organization, there should also be a cooperation between the team for who it's meant and the TM team.

> Also, beware:
> If the Policy is in docs, it SHOULD NOT BE EDITABLE by anybody!!!!
> It is an official policy.

I will ask Martijn Maandag who is helping with this, how this works.
Simply said, I can't even change documents on that website even though I have an account.

**Comment by infograf768** [comment]
Want me to try modifying?

**Comment by conconnl** [comment]
It's locked for Administrators

**Comment by wilsonge** [comment]
<img width="810" alt="Screenshot 2020-12-17 at 11 58 27" src="https://user-images.githubusercontent.com/1986000/102485201-2dd5f700-405f-11eb-8b29-bad8365df5a8.png">

I can confirm that

**Comment by conconnl** [comment]
The actions needed to move from CJO to DJO are done.
Doesn't mean that the information on docs is complete or updated.
Both pages will be updated, this is not specifically related to the component.

https://docs.joomla.org/Translations_Working_Group
https://docs.joomla.org/Translation_of_Language_Packs
